src/Microsoft.TemplateEngine.Edge/Settings/TemplateCache.cs

```diff
@@ -47,38 +47,6 @@ public TemplateCache(IEngineEnvironmentSettings environmentSettings, JObject par
         [JsonProperty]
         public IReadOnlyList<TemplateInfo> TemplateInfo { get; set; }
 
-        public IReadOnlyCollection<IFilteredTemplateInfo> List(bool exactMatchesOnly, params Func<ITemplateInfo, MatchInfo?>[] filters)
```

Comment:
Let's preserve this signature, call the new location of this method & mark it as obsolete (maybe?)